modules/mod-deep-learning/DeepModel.cpp

```diff
@@ -28,7 +28,11 @@ FilePath DeepModel::DLModelsDir()
 
 FilePath DeepModel::BuiltInModulesDir()
 {
-   return FileNames::MkDir( wxFileName( FileNames::BaseDir(), wxT("deeplearning-models") ).GetFullPath() );
+   #if defined(__WXOSX__) || defined(__WXMSW__)
+      return FileNames::MkDir( wxFileName( FileNames::BaseDir(), wxT("deeplearning-models") ).GetFullPath() );
+   #else
+      return FileNames::MkDir( wxFileName( FileNames::ResourcesDir(), wxT("deeplearning-models") ).GetFullPath() );
+   #endif
```

Comment:
is there a chance we could just use `ResourcesDir()` for all platforms?

Reply:
I think this would be ideal, but I'm still working on the windows build and can't test mac builds yet. 

we can either wait until it's tested on all platforms, or I can create issues to check this support when we have the builds working

merging now helps prevent a hacky solution to this problem on linux

Reply:
yeah, sounds good. merge away! let's open a new issue for using the same resource dir to not forget ab it.